examples/advanced/edge/jobs/cifar10_mobile_et/app/config/config_fed_server.json

```diff
@@ -3,14 +3,60 @@
     "num_rounds": 10,
     "workflows": [
         {
-            "id": "edge_executorch_controller",
-            "path": "nvflare.edge.controllers.edge_executorch_controller.EdgeExecutorchController",
+            "id": "sage_controller",
+            "path": "nvflare.edge.controllers.sage.ScatterAndGatherForEdge",
             "args": {
                 "num_rounds": "{num_rounds}",
-                "task_name": "cifar10",
+                "aggregator_id": "aggregator",
+                "persistor_id": "persistor",
+                "shareable_generator_id": "shareable_generator",
+                "assessor_id": "assessor"
+            }
+        }
+    ],
+    "components": [
+        {
+            "id": "aggregator",
+            "path": "nvflare.edge.aggregators.edge_json_accumulator.EdgeJsonAccumulator",
+            "args": {
+                "aggr_key": "data"
+            }
+        },
+        {
+            "id": "persistor",
+            "path": "nvflare.app_opt.pt.file_model_persistor.PTFileModelPersistor",
+            "args": {
+               "model": {
+                  "path": "nvflare.edge.models.model.Cifar10ConvNet",
+                  "args": {}
+               }
+            }
+        },
+        {
+            "id": "shareable_generator",
+            "path": "nvflare.edge.shareablegenerators.executorch_shareable_generator.ExecutorchShareableGenerator",
+            "args": {
+                "base_model_path": "nvflare.edge.models.model.Cifar10ConvNet",
+                "executorch_model_path": "nvflare.edge.models.model.TrainingNet",
                 "input_shape": [32, 3, 32, 32],
                 "output_shape": [32]
             }
+        },
+        {
+            "id": "assessor",
+            "path": "nvflare.edge.assessors.count.CountAssessor",
+            "args": {
+                "min_count": 16,
+                "max_count": 100,
+                "timeout": 3600.0
+            }
+        },
+        {
+            "id": "evaluator",
```

Comment:
How is the evaluator used? It's not referenced in any other components.

Reply:
evaluator is used to do a round of global model evaluation and record the performance to TB when a new global model is produced, so it is a standalone and acts when new global model event happens